Manifest.toml

```diff
@@ -0,0 +1,1138 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
I think `Manifes.toml` is accidentally committed here, need to remove it before you merge this PR.

Reply:
i had added Manifest when I was pointing MIRT to my branch of Wavelets (I thought only the Manifest has the specific branch info, though I could be wrong about that).  but now that I am using the main branch of Wavelets again, I agree I don't need Manifest.  thanks.